Directory.Build.props

```
@@ -21,6 +21,9 @@
     <IsAnalyzerProject Condition="!$(IsTestProject) and
                                   ($(MSBuildProjectName.EndsWith('.Analyzers')) or $(MSBuildProjectName.EndsWith('.Analyzers.CSharp')))"
                                   >true</IsAnalyzerProject>
+
+    <EnableTrimAnalyzer Condition="!$(IsTestProject) and !$(IsAnalyzerProject) and
+                                  !($(MSBuildProjectName.EndsWith('.TestUtilities'))">true</EnableTrimAnalyzer>
```

We should always set the default (i.e., without conditions) value, and after that set the value based on a condition. After that you shouldn't need to configure `EnableTrimAnalyzer` in other places....

Would this work?
```suggestion
    <EnableTrimAnalyzer>true</EnableTrimAnalyzer>
    <EnableTrimAnalyzer Condition="$(IsTestProject) or !$(IsAnalyzerProject) or $(MSBuildProjectDirectory.Contains('\tests\'))">false</EnableTrimAnalyzer>
```